### PR TITLE
Update lims.py

### PR DESCRIPTION
**Pull request by rikerl-scilifelab**

### CODE REVIEW

File: access/lims.py [hunk 1]
```diff
@@ -1,76 +1 @@
...
-        hit = node.text
-    return hit
-    
-  def getattribute(self, searchattribute, searchvalue, attribute):
-    r = requests.get(self.uri + searchattribute + '/' + searchvalue, auth=(self.user, self.pwd))
-    tree = ElementTree.fromstring(r.text.encode('utf8'))
-    hit = "No hit"
-    for node in tree:
-#      print node
-      for key in node.attrib:
```
Comment (rikerl-scilifelab) [inline access/lims.py:66]: It might not be very clever to iterate over nodes and attributes . . .

File: access/lims.py [hunk 2]
```diff
@@ -1,76 +1 @@
...
-      if len(line) > 3 and not line[0] == "#":
-        pv = line.rstrip().split(" ")
-        arg = pv[0]
-        pv.pop(0)
-        params[arg] = ' '.join(pv)
-  return params
-
-class limsconnect(object):
-  
-  def __init__(self, User, Passwd, baseuri):
```
Comment (robinandeer) [inline access/lims.py:27]: Capitalised variable names indicate classes in Python :)

File: access/lims.py [hunk 3]
```diff
@@ -1,76 +1 @@
...
-    self.pwd = Passwd
-    self.uri = baseuri
-    self.entrypoints = { 'samples': 'limsid', 'artifacts': 'limsid' }
-
-  def __enter__(self):
-    return self
-
-  def __exit__(self, exc_type, exc_val, exc_tb):
-    if exc_type:
-      print '__exit__(%s, %s, %s)' % (exc_type, exc_val, exc_tb)
```
Comment (robinandeer) [inline access/lims.py:38]: I've been transitioning over to use the format minilanguage which is the new Python standard. We should consider using it throughout our Python code or not, hehe!

File: access/lims.py [hunk 4]
```diff
@@ -1,76 +1 @@
...
-      
-  def getroot(self):
```
Comment (robinandeer) [inline access/lims.py:40]: Just missing some docstrings ^^

**Pull request closed without merging**